### PR TITLE
changes from sampling branch

### PR DESCRIPTION
**Pull request by cremebrule**

### CODE REVIEW

File: omnigibson/objects/dataset_object.py [hunk 1]
```diff
@@ -104,6 +104,10 @@ def __init__(
         load_config = dict() if load_config is None else load_config
         load_config["bounding_box"] = bounding_box
 
+        # TODO: Remove once meshes are fixed
+        from omnigibson.utils.bddl_utils import DO_NOT_REMESH_CLOTHS
+        load_config["remesh"] = model not in DO_NOT_REMESH_CLOTHS.get(category, set())
```
Comment (ChengshuLi) [inline omnigibson/objects/dataset_object.py:109]: change to BAD_CLOTH_MODELS and immediately raise an error if self.prim_type == PrimType.CLOTH and self.model in BAD_CLOTH_MODELS

File: omnigibson/scenes/scene_base.py [hunk 2]
```diff
@@ -237,6 +240,12 @@ def _load_objects_from_scene_file(self):
             obj = create_object_from_init_info(obj_info)
             # Import into the simulator
             og.sim.import_object(obj)
+            if isinstance(obj, DatasetObject) and obj.model in BOUNDING_CUBE_OBJECTS:
+                link_names = BOUNDING_CUBE_OBJECTS[obj.model]
+                for link_name in link_names:
+                    link = obj.links[link_name]
+                    for col_mesh in link.collision_meshes.values():
+                        col_mesh.set_collision_approximation("boundingCube")
```
Comment (ChengshuLi) [inline omnigibson/scenes/scene_base.py:248]: remove these changes - actually Cem fixed this model, so we should be okay.

File: omnigibson/utils/bddl_utils.py [hunk 3]
```diff
@@ -37,6 +38,87 @@
...
+    "pillowcase": {"dtoahb", "yakvci"},
+    "rubber_glove": {"leuiso"},
+    "scarf": {"kclcrj"},
+    "sock": {"vpafgj"},
+    "tank_top": {"fzldgi"},
+    "curtain": {"shbakk"}
+}
+
+DO_NOT_REMESH_CLOTHS = {}
+DO_NOT_REMESH_CLOTHS.update(BAD_MODELS)
```
Comment (ChengshuLi) [inline omnigibson/utils/bddl_utils.py:121]: unify these

**Review by ChengshuLi — APPROVED** [review]

File: omnigibson/objects/dataset_object.py (hunk 1 above)
Comment (cremebrule) [reply omnigibson/objects/dataset_object.py:109]: done

File: omnigibson/scenes/scene_base.py (hunk 2 above)
Comment (cremebrule) [reply omnigibson/scenes/scene_base.py:248]: done

**Comment by cgokmen** [comment]
Merging, will fix tests in follow up.

**Pull request merged**